File: Samples/iOS/iOS/Moment+JSON.m
//
//  Moment+JSON.m
//  CoreDataClientSample
//
//  Created by Nathaniel Potter on 11/4/14.
//  Copyright (c) 2014 Nathaniel Potter. All rights reserved.
//

#import "Moment+JSON.h"

@implementation Moment (JSON)

- (void)updateWithDictionary:(NSDictionary *)dictionary
{
    self.uid = dictionary[@"id"];
    self.kind = dictionary[@"kind"];
    self.title = dictionary[@"title"];
}

@end

File: Samples/iOS_Swift/CoreDataClientSample/Moment+JSON.swift
//
//  Moment+JSON.swift
//  CoreDataClientSample
//
//  Created by Nathaniel Potter on 1/31/15.
//  Copyright (c) 2015 Nathaniel Potter. All rights reserved.
//

import Foundation

extension Moment {
    override func updateWithDictionary(dictionary: [NSObject : AnyObject]!) {
        self.uid = dictionary["id"] as? String
        self.kind = dictionary["kind"] as? String
        self.title = dictionary["title"] as? String
    }
}